src/ProgressOnderwijsUtils/Pocos/BulkInsertTarget.cs

```diff
@@ -5,20 +5,23 @@ namespace ProgressOnderwijsUtils;
 
 public sealed class BulkInsertTarget
 {
+    public enum ReadOnlyTargetMode { NotAllowed, Allowed, }
+
     public const SqlBulkCopyOptions DefaultOptionsCorrespondingToInsertIntoBehavior = SqlBulkCopyOptions.CheckConstraints | SqlBulkCopyOptions.FireTriggers | SqlBulkCopyOptions.KeepNulls;
     public readonly string TableName;
     public readonly ColumnDefinition[] Columns;
     public readonly BulkCopyFieldMappingMode Mode;
     public readonly SqlBulkCopyOptions Options;
+    public readonly ReadOnlyTargetMode ReadOnlyTarget;
```

Comment:
Ik heb het idee ook hier dat "ReadOnlyTarget niet heel veel zegt.  SilentlySkipReadonlyTargetColumns oid en een-op-een doorsluizen is niet duidelijker?

Reply:
En nog een optie is dat we dit aan de BulkCopyFieldMappingMode enum toevoegen en dat een flags enum maken, al maak je't dan ook mogelijk om bijv. de combo van extra kolommen aan source _en_ dest te negeren, wat vermoedelijk technisch kan maar semantisch niet een typisch verstandige keuze is.